runtime/src/core/runner.js

```diff
@@ -25,15 +24,82 @@ export class GrainRunner {
     this._checkMemory = makeMemoryChecker(this);
     this.limitMemory = opts.limitMemory || -1;
     this.postImports = () => {};
+    this.encoder = new TextEncoder("utf-8");
+    this.decoder = new TextDecoder("utf-8");
+    const printNames = {};
+    const this_ = this;
```

Comment:
This shouldn't really be necessary. Is this to try to skirt around some `.bind` somewhere?

Reply:
Fixed. Used the wrong syntax below.